packages/shared/src/components/search/SearchPanel/SearchPanelSourceSuggestions.tsx

```diff
@@ -0,0 +1,108 @@
+import classNames from 'classnames';
+import React, { ReactElement, useContext } from 'react';
+import { useRouter } from 'next/router';
+import { SearchProviderEnum, SearchSuggestion } from '../../../graphql/search';
+import { useSearchProviderSuggestions } from '../../../hooks/search';
+import { SearchPanelContext } from './SearchPanelContext';
+import { useSearchPanelAction } from './useSearchPanelAction';
+import { LogEvent, Origin, TargetType } from '../../../lib/log';
+import LogContext from '../../../contexts/LogContext';
+import { webappUrl } from '../../../lib/constants';
+import { ButtonProps } from '../../buttons/Button';
+import { SearchPanelItem } from './SearchPanelItem';
+
+export type SearchPanelSourceSuggestionsProps = {
+  className?: string;
+  title: string;
+};
+
+type PanelItemProps = Pick<ButtonProps<'a'>, 'onClick'> & {
+  suggestion: SearchSuggestion;
+};
+
+const PanelItem = ({ suggestion, ...rest }: PanelItemProps) => {
+  const Icon = () => (
+    <img
+      loading="lazy"
+      src={suggestion.image}
+      alt={`${suggestion.title} logo`}
+      className="size-7 rounded-full"
+    />
```

Comment:
Use our `Image` component?

Reply:
Ill check why i opted to use this in source page, feel it had some reason but dont remember 😂